content/news/2022-09-02-gsoc-final-blog-david-chocholaty.md

```diff
@@ -0,0 +1,714 @@
+With that, the other extensional options can be used. Primarily, it allows
+for the user to use a much wider range, than the Pitch Shifter
+for the deck player. Due to the produced latency based on the pitch processing,
+the production delay has to be handled for the wet/dry or wet+dry mode
+to ensure that the original (dry) and processed (wet) signals overlap.
+
+#### Background
+List of terms from the field of music, sound processing
+and development in general:
+
+* Scale
+    * In music theory, a scale is any set of musical notes ordered
+     by fundamental frequency or pitch. The scale ordered
+     by increasing pitch is an ascending scale, and a scale ordered
+     by decreasing pitch is a descending scale[^2].
+* Octave
+    * In music, an octave is an interval between one musical pitch
+     and another with double its frequency[^3].
+* Interval
+    * In music theory, an interval is a difference in pitch between
+     two sounds. In western scales, intervals are most commonly differences
+     between whole tones and semitones[^4].
+* Semitone
+    * A semitone is a distance in pitch between a note
+     and the very next note, higher or lower. It is the smallest interval
+     in most western scales[^5].
+* Chromatic scale
+    * A chromatic scale is a set of twelve pitches used
+     in tonal music, with notes separated by the interval
+     of a semitone[^6].
+* Pitch
+    * Pitch is a perceptual property of sounds that allows
+     their ordering on a frequency-related scale. The pitch
+     is the quality that makes it possible to judge sounds as "higher"
+     and "lower" in the sense associated with musical melodies[^7].
+* Wet and dry signals
+    * Dry sound signals refer to the raw or unprocessed sounds
+     that usually come from a direct recording. On the other hand,
+     wet sounds refer to the processed sound/signal[^8].
+* Ring buffer (circular buffer)
+    * In computer science, a circular buffer or ring buffer
+     is a data structure that uses a single, fixed-size buffer
+     as if it were connected end-to-end. This structure lends itself
+     easily to buffering data streams[^9].
+* Audio buffer
+    * An audio buffer holds a fixed size amount of sampled audio data.
+     The audio buffer size determines the time allowed for the computer
+     to process the audio data. Thus, it also determines Latency.
+* Latency
+    * In the audio world, “latency” is another word for “delay”[^10].
+     The latency (time) of an audio system refers to the time difference
+     from the moment a signal is fed into the system, to the moment it appears
+     at the output[^11]. For example, audio latency is when there’s a noticeable
+     delay between the sound being played and the moment it reaches
+     the speakers[^10]. Depending on the application, such a delay can have various
+     effects. Usually, the aim is to achieve the lowest possible latency[^11].
+
```

Comment:
`10` and `11` are here twice, is that on purpose?

Reply:
Yes, it was meant to be. The reason for that is, that the description is a combination of citations from two sources and the whole paragraph made more sense to me if the sentences have combined these ways, so the two citation sources alternate.